pkg/intermediate/aggregate.go

```diff
@@ -645,9 +693,122 @@ func (a *AggregationProcess) addFieldsForStatsAggregation(record entities.Record
 			}
 		}
 	}
+
+	// Initialize the throughput elements.
+	var timeStart, timeEnd uint32
+	var byteCount, reverseByteCount uint64
+	timeStart, err := getUnsigned32ValueByIeName(record, "flowStartSeconds")
+	if err != nil {
+		return err
+	}
+	timeEnd, err = getUnsigned32ValueByIeName(record, "flowEndSeconds")
+	if err != nil {
+		return err
+	}
+	byteCount, err = getUnsigned64ValueByIeName(record, "octetTotalCount")
+	if err != nil {
+		return err
+	}
+	reverseByteCount, err = getUnsigned64ValueByIeName(record, "reverseOctetTotalCount")
+	if err != nil {
+		return err
+	}
+	incomingVal := byteCount / (uint64(timeEnd - timeStart))
+	reverseIncomingVal := reverseByteCount / (uint64(timeEnd - timeStart))
+	throughputVals := []uint64{incomingVal, reverseIncomingVal}
+
+	for i, element := range antreaThroughputElements {
```

Comment:
I re-wrote the code where we initialize throughput elements, grouping the throughput elements by common fields/src fields/ dst fields. Does the current format make more sense to you? @zyiou